emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java

```diff
@@ -23,6 +19,10 @@ public class ConceptsToEClassesMapping {
   private final Map<Concept, EClass> conceptsToEClasses = new HashMap<>();
   private final Map<Interface, EClass> interfacesToEClasses = new HashMap<>();
 
+  public ConceptsToEClassesMapping() {
+    populateInternal();
```

Comment:
Should we do this by default, or only on explicit call?

Reply:
By default I would enable it, but offer an alternative constructor with a flag to control it